examples/plot_connectivity.py

```diff
@@ -8,13 +8,16 @@
 
 # Author: Nick Tolley <nicholas_tolley@brown.edu>
 
+# sphinx_gallery_thumbnail_number = 4
```

Comment:
should we move this example before the MPI example? I think that makes more sense logically